### PR TITLE
updates the readme with some changes

### PR DESCRIPTION
**Pull request by raghuganapathyUCR**
updates the readme with some changes - new content and refactor old content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -227,30 +238,45 @@ See the format of the config file below with sample values:
...
+Provide the path to the configuration file above with the `-p,--path` argument to `core.jar`, and no other flags are needed.
+
+To view descriptions of all flags, simply run the JAR with the `--help` option.
+
+## NullAway Compatibility
+
+- `Annotator` version `1.3.6` is compatible with `NullAway` version `0.10.10` and above.
+
+
+## Features 🧱
```
Comment (nimakarimipour) [inline README.md:254]: Maybe we can delete emojis for now :)

File: README.md [hunk 2]
```diff
@@ -227,30 +238,45 @@ See the format of the config file below with sample values:
...
+
+- Automatically infers nullability types and injects corresponding annotations
+- Injects suppression annotations for unresolved NullAway errors
+- Minimizes the number of reported NullAway errors
+- Reduces manual effort when applying NullAway to build systems
+
+<!-- ## Limitations ⏹️
```
Comment (nimakarimipour) [inline README.md:261]: See comment above.

File: README.md [hunk 3]
```diff
@@ -227,30 +238,45 @@ See the format of the config file below with sample values:
...
+
+Annotator is designed to provide an automated solution for annotating source code to pass NullAway checks. However, there are certain limitations:
+
+- Annotator may not always infer the correct nullability type for every variable and method, which may require manual review and adjustment.
+- Annotator does not resolve runtime exceptions related to nullability; it only focuses on satisfying NullAway checks.
+
+<!-- Annotator is not intended as a replacement for thorough code review and testing. -->
+need to ask nima to add more here.. -->
+## Contributing 🤝
```
Comment (nimakarimipour) [inline README.md:270]: See comment above.

**Review by nimakarimipour — CHANGES_REQUESTED** [review]
This mostly looks good to me, except for a minor comment.

File: README.md (hunk 1 above)
Comment (raghuganapathyUCR) [reply README.md]: Updated it, removed all the emojis (Resolving all comments, please resolve this one after review)

**Review by nimakarimipour — APPROVED** [review]

**Pull request merged**